public/index.html

```diff
@@ -1,30 +1,42 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Linkbrary</title>
+    <meta property="og:title" content="Linkbrary" key="title" />
```

Comment:
```suggestion
    <meta property="og:title" content="Linkbrary"/>
```

여기서 `key` 프로퍼티는 없는 속성으로 보입니다 ! 
[mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) 문서를 참고하였습니다 =)